tests/common/snappi/qos_fixtures.py

```diff
@@ -1,108 +0,0 @@
-import pytest
-
-"""
```

Comment:
Why do you remove this file?

Reply:
this has been moved to a separate PR https://github.com/sonic-net/sonic-mgmt/pull/8186/

Reply:
We should fix this in this PR.